include/LoRa.h

```diff
@@ -0,0 +1,22 @@
+#ifndef LORA_H
+#define LORA_H
+
+#include <stdint.h>
+
+// LoRa
+void LoRa_Tx_Init(void);
+void Set_Frequency_LoRa(void);
+
+void SX1276Write(uint16_t addr, uint8_t data);
```

Comment:
is het nodig dat al deze functies public zijn? zo ja, plaats er een comment bij.